openmicroscopy/ome-help
# Export options

Fix the matrix of export options and add support for Plates.
See https://trello.com/c/bxkT6Jla/107-export-ome-tiff-export-disabled-in-web-when-available-from-cli

To test:
 - Check that every cell of the table corresponds to the actual ability to download/export data as described.
 - Table looks OK? (replaced image of table with actual html table).

Old vv new tables:

![screen shot 2017-09-13 at 13 51 44](https://user-images.githubusercontent.com/900055/30378341-cab6458c-988a-11e7-9277-2b561469c450.png)
